client/my-sites/site-settings/settings-performance/controller.js

```diff
@@ -0,0 +1,19 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import PerformanceMain from 'my-sites/site-settings/settings-performance/main';
```

Comment:
We could shorten `'my-sites/site-settings/settings-performance/main'` to just `./main`.

Reply:
✅https://github.com/Automattic/wp-calypso/pull/29597/commits/de2eaa7a61f653b1fadbe62c5b05a21e7a526f5c